docs/examples/search_vector_similarity_examples.ipynb

```diff
@@ -98,7 +453,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.2"
+   "version": "3.9.12"
```

Comment:
Is there something specific here that requires *a user* to upgrade to 3.9.12 in ipython land? If so, I missed it - please share!